packages/hash/api/src/db/postgres/link/getEntityOutgoingLinks.ts

```diff
@@ -27,29 +26,18 @@ export const getEntityOutgoingLinks = async (
   params: {
     accountId: string;
     entityId: string;
-    entityVersionId?: string;
+    activeAt?: Date;
     path?: string;
   },
 ) => {
-  const rows = await conn.any(sql<DBLinkRow>`
-    select ${mapColumnNamesToSQL(linksColumnNames)}
-    from links
-    where
-    ${sql.join(
-      [
-        sql`source_account_id = ${params.accountId}`,
-        sql`source_entity_id = ${params.entityId}`,
-        params.entityVersionId !== undefined
-          ? sql`${params.entityVersionId} = ANY(source_entity_version_ids)`
-          : [],
-        params.path !== undefined ? sql`path = ${params.path}` : [],
-      ].flat(),
-      sql` and `,
-    )}
-    order by index
-  `);
+  const dbLinks = await getLinks(conn, {
```

Comment:
The `getEntityOutgoingLinks` is just a wrapper around `getLinks`. 
I think it's good that we have a specific name for doing the `getEntityOutgoingLinks` query, but are there other reasons why we don't use `getLinks` directly?

Reply:
Also noticed that I forgot to get rid of `getEntityWithOutgoingEntityIds` in that file, will also remove that.

Reply:
Sorry, I had to rewrite this PR several times as I was figuring out the best approach, I'm realising now the structure of this ended up a bit too verbose in the final state.

Reply:
No worries! :)

Reply:
Addressed in https://github.com/hashintel/hash/pull/364/commits/63928ad84880574da002dcbd58b52197051d4dfa

(I also realised the documentation of that method is essentially a duplicate of the documentation in the adapter class, so have removed it in https://github.com/hashintel/hash/pull/364/commits/bc51dd79d0289c6ef1989438cc1d3b1d77795020)